SRC/Screens/AquariumCalculate.js

```diff
@@ -11,12 +13,21 @@ const AquariumCalculate = ({route, navigation}) => {
   }, []);
   return (
     <View style={styles.body}>
-      <Pressable
-        onPress={() => {
-          handleCalculate({height: 5, lenght: 10, width: 20});
-        }}>
-        <Text>Calculate: </Text>
-      </Pressable>
+      {isEnabled ? (
+        <Text style={styles.isFalse}>Metric</Text>
+      ) : (
+        <Text style={styles.isTrue}>Metric</Text>
+      )}
+      <UnitSwitch
+        style={styles.switchBtn}
+        setIsEnabled={setIsEnabled}
+        isEnabled={isEnabled}
+      />
+      {isEnabled ? (
+        <Text style={styles.isTrue}>Imperial</Text>
+      ) : (
+        <Text style={styles.isFalse}>Imperial</Text>
```

Comment:
There is no need to have this text here. Move it to the UnitSwitch component. Same as above